src/Neo/Wallets/Helper.cs

```diff
@@ -90,78 +104,90 @@ internal static byte[] XOR(byte[] x, byte[] y)
         /// <returns>The network fee of the transaction.</returns>
         public static long CalculateNetworkFee(this Transaction tx, DataCache snapshot, ProtocolSettings settings, Func<UInt160, byte[]> accountScript, long maxExecutionCost = ApplicationEngine.TestModeGas)
```

Comment:
This will make conflicts into my never ending pull request https://github.com/neo-project/neo/pull/3385 :'(

Reply:
We may merge #3385 as it is now and move non-standard verification scripts support to another issue. I think it's acceptable way since #3385 contains everything that's required except non-standard verification scripts handling.

Reply:
Moved to https://github.com/neo-project/neo/issues/3539.